src/utils/ellipse_formatter.jl

```diff
@@ -123,7 +123,7 @@ function parametric2quad(semiaxis_lengths; center=[0 0], ccw_angle=0)
     return S, vec(center)
 end
 
-function quad2parametric(S; center=[0 0])
+function quad2parametric(S, center=[0 0])
```

Comment:
Isn't a semicolon necessary to specify keyword args?

Reply:
Yeah, my mistake, I was thinking that this was causing the trouble